CHANGELOG.md

```diff
@@ -3,15 +3,23 @@
 ## Unreleased
 
 ### Added
+
 - Documentation for Logger API Requests in `docs/api_requests/logger.md`.
 
 ### Changed
+
 - Updated the swagger definition of the `Logger` to specify the required fields
   and provide default values for optional fields.
+- Default `seccomp-level` is `2` (was previously 0).
```

Comment:
nit: specify that this applies for running outside the jailer (which has been setting it to 2 for a while)

Reply:
No need, either way you run, it is now 2.